spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jasmine */
+/* global GOVUK */
+
+describe('The intervention component', function () {
+  const FIXTURE = `
+    <section class="gem-c-intervention" role="region" data-module="intervention">
+      <h2>
+        <a href="/next-steps-for-your-business">
+          Check the next steps for your limited company
+        </a>
+      </h2>
+
+      <p class="govuk-body"></p>
+      <p class="govuk-body"></p>
+    </section>`
+
+  function loadInterventionComponent () {
+    var element = document.querySelector('[data-module="gem-intervention"]')
+    new GOVUK.Modules.Intervention(element).init()
+  }
+
+  beforeEach(function () {
+    spyOn(GOVUK.analytics, 'trackEvent')
+    window.setFixtures(FIXTURE)
+  })
+
+  afterEach(function () {
+    if (GOVUK.analytics.trackEvent.calls) {
+      GOVUK.analytics.trackEvent.calls.reset()
+    }
+  })
+
+  it('triggers a tracking event when the component is displayed', function () {
+    loadInterventionComponent()
+    expect(GOVUK.analytics.trackEvent.calls.count()).toEqual(1)
```

Comment:
This is good, but you can also make this test more specific e.g.

`expect(GOVUK.analytics.trackEvent).toHaveBeenCalledWith('interventionBanner', 'interventionShown')`